packages/clay-multi-select/src/htmlEscape.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-control-regex */
+const AMP_RE_ = /&/g;
+const LT_RE_ = /</g;
+const GT_RE_ = />/g;
+const QUOT_RE_ = /"/g;
+const SINGLE_QUOTE_RE_ = /'/g;
+const NULL_RE_ = /\x00/g;
+const ALL_RE_ = /[\x00&<>"']/;
+
+// All credits from this code to https://github.com/google/closure-library/blob/master/closure/goog/string/internal.js#L173
```

Comment:
This is probably insufficient from a legal perspective. That code is under the Apache License, which requires you to retain the copyright notices that came with it.